### DISCUSSION TITLE
Why were the patch versions for CVE-2021-36157 released so late?

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [cortexproject/cortex#5470] [Q&A] [answered]
Hello, we are a research team working on Golang. During our investigation, we found CVE-2021-36157 (snyk reference https://security.snyk.io/vuln/SNYK-GOLANG-GITHUBCOMCORTEXPROJECTCORTEXPKGTENANT-1536565) was addressed in commit d9e1f81f40c607b9e97c2fc6db70ae54679917c4. However, we noticed that the patch version ([v1.11.0-rc.0](https://github.com/cortexproject/cortex/releases/tag/v1.11.0-rc.0)) was released after long time (74 days).  We are curious about the reasons behind the delayed release of the patch version, as it may hinder the efficient distribution of patches to downstream users. Could the reason be 

1.Issues with testing and CI checking. 

2.Other commits have to be incorporated into one release. 

3.By convention, versions are not frequently released. 

4.Other reasons. 

Thank you for your attention, and we look forward to receiving your reply.

**Comment by friedrichg** [comment, marked as answer]
every commit is released to quay.io and docker hub. So don't think users had to wait to use the patched version.
We also publish releases that are more tested every few months.
